src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -178,7 +178,6 @@ public static TypedData ToRpc(this object value, ILogger logger, Capabilities ca
                     }
 
                     request.Body.Position = 0;
-                    http.Body = body.ToRpc(logger, capabilities);
```

Comment:
Unless this is driven by worker capabilities, it will introduce a change that will require language worker updates. Ideally, this would be done in a non-breaking, backwards compatible way (i.e. the capability needs to be advertised by the worker)

Reply:
I have suggested @yojagad to first try with other languages as this issue is very likely just with nodejs worker. Other language workers might not be relying on the duplicates.